config/globalVariable.js

```diff
@@ -111,6 +110,86 @@ module.exports = function () {
     }
   });
 
+  global.sessions = {};
+
+  let categories = new Promise(async function(resolve, reject) {
```

Comment:
@aman-tunerlabs - Let's use library categories helper and not write DB queries here.